src/GearScalingOverWrite.h

```diff
@@ -24,19 +24,61 @@ extern "C" float GetGearScaling(cube::Item * item, cube::Creature* creature, int
 	float base_res = ((base * 0.5f) / (float)0x20);
 	float mod_modifier = (mod3 / 0x10624DD3) / 7.0f;
 
-	float X = 2;
+	float X = 1.37;
 	float Y = base_res + effective_rarity + mod_modifier;
 
 	float result = std::powf(X, Y);
 
 	cube::Game* game = cube::GetGame();
 	if (game && creature->entity_data.hostility_type == cube::Creature::EntityBehaviour::Player)
 	{
-		result *= 1 + 0.05f * GetItemLevel(item);
+		//result *= 1 + 0.05f * GetItemLevel(item);
+		result *= 0.2 * std::pow(2.7183, 0.2 * GetItemLevel(item)) / 1.21 * std::pow(0.99, 1 + 0.07 * GetItemLevel(item) * GetItemLevel(item));
```

Comment:
Yes this is a good formula, but we will have to test if this does not affect performance too much